deepmd/dpmodel/output_def.py

```diff
@@ -142,6 +144,10 @@ class OutputVariableCategory(IntEnum):
     """Virial, the transposed negative gradient with cell tensor times cell tensor, see eq 40 JCP 159, 054801 (2023). """
     DERV_R_DERV_R = OutputVariableOperation.DERV_R | OutputVariableOperation._SEC_DERV_R
     """Hession matrix, the second derivative w.r.t. coordinates."""
+    DERV_R_MAG = OutputVariableOperation.DERV_R | OutputVariableOperation.MAG
+    """Magnetic part of negative derivative w.r.t. coordinates. (e.g. magnetic force)"""
+    DERV_C_MAG = OutputVariableOperation.DERV_C | OutputVariableOperation.MAG
+    """Magnetic part of atomic component of the virial."""
```

Comment:
Not used yet, are these necessary for `force_mag` and `atom_virial` to be added into `apply_operation`? @njzjz

Reply:
Not necessary, `DERV_C_MAG` is just a shortcut for `OutputVariableOperation.DERV_C | OutputVariableOperation.MAG` (their values are the same).